indexers/query/AnnoyQueryIndexer/__init__.py

```diff
@@ -0,0 +1,59 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Tuple
+
+import numpy as np
+from jina.executors.indexers.query.vector import NumpyQueryIndexer
+
+
+class AnnoyQueryIndexer(NumpyQueryIndexer):
+    """Annoy powered vector indexer
+
+    For more information about the Annoy supported parameters, please consult:
+        - https://github.com/spotify/annoy
+
+    .. note::
+        Annoy package dependency is only required at the query time.
+    """
+
+    def __init__(self, metric: str = 'euclidean', n_trees: int = 10, search_k: int = -1, *args, **kwargs):
```

Comment:
I would not have it inherit from NumpyQueryIndexer if possible.

Even more now with 2.0 coming where no inter executor dependency is desired.